### PR TITLE
GRANITE-31511 - Allow to specify the replication agent name when acti…

### PR DESCRIPTION
**Pull request by cschneider**
…vating a page

### CODE REVIEW

File: src/main/java/com/adobe/cq/testing/client/ReplicationClient.java [hunk 1]
```diff
@@ -78,6 +78,26 @@ public SlingHttpResponse activate(String nodePath, int... expectedStatus) throws
                     .build(),
                 HttpUtils.getExpectedStatus(SC_OK, expectedStatus));
     }
+    
+    /**
+     * Activates (publish) a node using a specific agent.
+     *
+     * @param nodePath path of the node to activate
```
Comment (volteanu) [inline src/main/java/com/adobe/cq/testing/client/ReplicationClient.java:85]: missing javadoc for `agentId`

File: src/main/java/com/adobe/cq/testing/client/ReplicationClient.java [hunk 2]
```diff
@@ -96,6 +116,25 @@ public SlingHttpResponse deactivate(String pagePath, int... expectedStatus) thro
                     .build(),
                 HttpUtils.getExpectedStatus(SC_OK, expectedStatus));
     }
+    
+    /**
+     * Deactivates (un-publish) a node using a specific agent.
+     *
+     * @param pagePath path of the node to deactivate
```
Comment (volteanu) [inline src/main/java/com/adobe/cq/testing/client/ReplicationClient.java:123]: missing javadoc for `agentId`

**Review by volteanu — APPROVED** [review]
LGTM!

**Comment by dulvac** [comment]
closed as this was superseded by other PRs

**Pull request closed without merging**